src/lib/api-docs/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export { fetchCloudApiVersionData } from './fetch-cloud-api-version-data'
```

Comment:
I've moved all the `api-docs` utils I was previously "cross importing" from the "old view" to the "new view" up to `lib`. 

Detangling the two views is meant to add clarity to what the new view consumes, and should also make it easier to fully deprecate and remove the old view code once we're ready.